README.md

```diff
@@ -0,0 +1,92 @@
+# Contributing
```

Comment:
It would be great if you could add some information about the project which I gave in the issue description.
Tell me if you are able to do that, If not I will merge this pr else I will allow you to add information and then merge it.

Reply:
Will work on it. Thanks for the suggestions.